SimCalorimetry/CastorSim/plugins/CastorHitAnalyzer.h

```diff
@@ -7,8 +7,8 @@
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
```

Comment:
This file was included in CastorHitAnalyzer.cc, and it should therefore get removed